cmd/rig/cmd/capsule/common.go

```diff
@@ -283,22 +293,154 @@ func DeployAndWait(
 	if err != nil {
 		return err
 	}
-	fmt.Println("Deploying to capsule", capsuleName)
+	fmt.Println("Deploying to capsule", capsuleID)
 	if noWait {
 		return nil
 	}
-	return WaitForRollout(ctx, rig, scope, capsuleName, revision, rolloutID, timeout, rollbackID)
+	return WaitForRollout(ctx, rig, projectID, environmentID, capsuleID, revision, rolloutID, timeout, rollbackID)
+}
+
+func DeployDry(
+	ctx context.Context, rig rig.Client, projectID, environmentID, capsuleID string, changes []*capsule.Change,
+	currentRolloutID uint64, currentFingerprint *model.Fingerprint,
+	scheme *runtime.Scheme, isInteractive bool,
+) error {
```

Comment:
We need to wrap all these arguments in an `Input` struct. Same for the other deploy commands.
It doesn't have to be now as we need to do it for old functions as well. But some day..